include/openthread/thread.h

```diff
@@ -73,6 +73,17 @@ typedef enum
     OT_DEVICE_ROLE_LEADER   = 4, ///< The Thread Leader role.
 } otDeviceRole;
 
+#define OT_DOMAIN_NAME_MAX_SIZE 16 ///< Maximum size of the Thread Domain Name field (bytes).
+
+/**
+ * This structure represents a Domain Name.
+ *
+ */
+typedef struct otDomainName
```

Comment:
Is there plan to use this in public API?

Reply:
good question. not so far. will remove it.

Reply:
removed.